libcontainer/factory_linux.go

```diff
@@ -198,9 +198,6 @@ func (l *LinuxFactory) Create(id string, config *configs.Config) (Container, err
 	if err := os.MkdirAll(containerRoot, 0o711); err != nil {
 		return nil, err
 	}
-	if err := os.Chown(containerRoot, unix.Geteuid(), unix.Getegid()); err != nil {
-		return nil, err
-	}
```

Comment:
Could there be cases where `containerRoot` already exists? (in which case `os.MkdirAll()` is  a no-op, and the directory may not have the correct permissions?

Reply:
@thaJeztah see above (line 158) -- if the directory exists, this function errors out.

Reply:
IOW the last component of `containerRoot` is guaranteed to not exist before MkdirAll

Reply:
> see above (line 158) -- if the directory exists, this function errors out.

Ah, thanks! Looks like I didn't scroll up far enough 👍